go/felix/iptables/table.go

```diff
@@ -531,6 +533,14 @@ func (t *Table) Apply() {
 				failedAtLeastOnce = true
 				continue
 			} else {
+				t.logCxt.WithError(err).Error("Failed to program iptables, loading diags before panic.")
+				cmd := t.newCmd(t.iptablesSaveCmd, "-t", t.Name)
+				output, err2 := cmd.Output()
+				if err2 != nil {
+					t.logCxt.WithError(err2).Error("Failed to load iptables state")
+				} else {
+					t.logCxt.WithField("iptablesState", string(output)).Error("Current state of iptables")
+				}
```

Comment:
I believe it'll use `\n`.  That's what we get elsewhere.  Is that nicer or worse than the python logging?  It has the advantage of being easier to machine parse.